siteapp/urls.py

```diff
@@ -11,19 +11,15 @@
 from .settings import *
 
 urlpatterns = [
-    url(r"^$", views.homepage, name="homepage"),
-    url(r"^login$", views.homepage, name="homepage"),
-    url(r"^(privacy|terms-of-service|love-assessments)$", views.shared_static_pages),
+    url(r"^(?![\s\S]|login)$", views.homepage, name="homepage"),
+    url(r"^(privacy|terms-of-service|love-assessments)$", views.shared_static_pages, name="privacy_terms_love"),
```

Comment:
I have no clue how these seemingly unrelated items all use the same shared view and pages, but hey... that's out of scope of this PR.

Reply:
good question so the view shared_static_pages uses the page name as an arg so the url tag in the template is looking for a page arg and the url pattern is only supplying one either privacy or terms-of-service or love-assessments)